src/murfey/server/__init__.py

```diff
@@ -96,6 +98,11 @@ def run():
         type=int,
         default=8000,
     )
+    parser.add_argument(
+        "--transport",
+        help="Transport type for Zocalo connection (default: Pika Transport)",
+        default="PikaTransport",
+    )
```

Comment:
why not `workflows.transport.add_command_line_options(parser, transport_argument=True)`?